src/console.php

```diff
@@ -13,11 +13,10 @@
     ->register('my-command')
     ->setDefinition(array(
         // new InputOption('some-option', null, InputOption::VALUE_NONE, 'Some help'),
-    ))
+    )
+    )
```

Comment:
jak już to powinno byc )) w jednej lini ale cale $console->register('my-command') mozna wywalic bo nic nie robi, jest niepotrzebne